src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -484,7 +484,9 @@ private List<String> validateCourtTextOld(CaseData caseData) {
         List<String> errorsMessages = new ArrayList<>();
         // Tactical fix. We have an issue where null courtLocation is being submitted.
         // We are validating it exists on submission if not we return an error to the user.
-        if (caseData.getCourtLocation() == null || caseData.getCourtLocation().getApplicantPreferredCourt() == null) {
+        if (caseData.getCourtLocation() == null
```

Comment:
this needs to be removed, it's there to check whether getApplicantPreferredCourt (3 digit value) is there for prod